projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/datajobs/DataJobsDeploymentController.java

```diff
@@ -135,7 +145,10 @@ public ResponseEntity<Void> deploymentUpdate(
             sendNotification,
             operationContext.getUser(),
             operationContext.getOpId());
-
+        if (dataJobDeploymentPropertiesConfig.getWriteTos().contains(WriteTo.DB)) {
```

Comment:
Somewhere else I expressed my dislike for those "ifs" I think . We should sue factory and strategy pattern instead. But that's beyond this change.